pkgs/development/tools/qtcreator/default.nix

```diff
@@ -23,7 +23,7 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ qmake makeWrapper ];
 
-  patches = optional (stdenv.isAarch64 || stdenv.isAarch32) ./0001-Fix-Allow-qt-creator-to-build-on-arm-aarch32-and-aar.patch;
+  patches = optional stdenv.hostPlatform.isArm ./0001-Fix-Allow-qt-creator-to-build-on-arm-aarch32-and-aar.patch;
```

Comment:
It looks like isArm is a synonym for isAarch32, so this doesn't apply the patch on aarch64 when it should.